package.json

```diff
@@ -3,7 +3,7 @@
     "test": "xo && mocha"
   },
   "devDependencies": {
-    "chrome-stub": "^1.2.0",
+    "sinon-chrome": "^1.1.2",
```

Comment:
Just curious, why this change? It appears the usage is exactly the same in the tests. Are you planning on adding more to the tests that chrome-stub doesn't support?